wallet/src/lib.rs

```diff
@@ -330,15 +337,13 @@ mod wallet {
     /// Send cycles to another canister.
     #[update(guard = "is_custodian_or_controller", name = "wallet_send")]
     async fn send(args: SendCyclesArgs) -> Result<(), String> {
-        match api::call::call_with_payment(args.canister.clone(), "wallet_receive", (), args.amount)
-            .await
-        {
+        match api::call::call_with_payment(args.canister, "wallet_receive", (), args.amount).await {
```

Comment:
Do we need to pass `None` for the parameter to `wallet_receive`?

Reply:
No, Candid subtyping handles that automatically.